resources/assets/js/app/services/NotificationService.js

```diff
@@ -0,0 +1,13 @@
+import {securedRequest} from './RequestService';
+
+export const getNotifications = () => {
+    return securedRequest.get('/api/v1/notifications');
+};
+
+export const markAsRead = (id) => {
+    return securedRequest.patch(`/api/v1/notifications/${id}`);
```

Comment:
patch request requires fields that you're going to update. Replace this request with PUT and send it to /api/v1/notifications/${id}/status with body {status: ${someStatus}}